tests/test_utils.py

```diff
@@ -0,0 +1,41 @@
+import os
+from autowebcompat import utils
+
+
+def test_mkdir():
+    direc_path = os.path.dirname(os.path.realpath(__file__)) + "test"
+    utils.mkdir(direc_path)
+    assert(os.path.exists(direc_path))
```

Comment:
You can assert that this is a directory.
Also, you can test calling it again to check that it doesn't fail when the directory already exists.